content/en/mobile_testing/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Mobile Application Testing and Monitoring
+kind: documentation
+description: "Create intelligent, self-maintaining mobile tests to ensure the most critical parts of your mobile applications are up and running from real devices."
+disable_sidebar: true
+further_reading:
+- link: "https://www.datadoghq.com/blog/test-creation-best-practices/"
+  tag: "Blog"
+  text: "Best practices for creating end-to-end tests"
+- link: "/mobile_testing/mobile_app_tests"
+  tag: "Documentation"
+  text: "Learn how to create Synthetic mobile app tests"
+- link: "/mobile_testing/settings"
+  tag: "Documentation"
+  text: "Learn how to upload your iOS or Android mobile applications"
+- link: "/continuous_testing/"
+  tag: "Documentation"
+  text: "Learn about Continuous Testing & CI/CD"
+cascade:
+  algolia:
+    tags: ['mobile_testing']
+---
+
+Mobile Application Testing allows you to test and monitor key business flows for Android and iOS applications using real devices. 
```

Comment:
This feels a little short, can we update it to this:

> Datadog Mobile Application Testing enables you to create and run end-to-end synthetic tests for applications that run on both iOS and Android devices. 
> 
> Datadog runs these tests on real devices to provide a realistic, step-by-step representation of key application workflows, screenshots of each step, and detailed pass/fail results so that your team can quickly visualize what went wrong.

Reply:
```suggestion
Mobile Application Testing allows you to test and monitor key business flows for Android and iOS applications using real devices. Datadog runs these tests on real devices to provide a realistic, step-by-step representation of key application workflows, screenshots of each step, and detailed pass or fail results so your team can quickly visualize what went wrong.
```